apps/patient/src/views/Pharmacy.tsx

```diff
@@ -446,20 +459,23 @@ export const Pharmacy = () => {
     }
   }, [enableOpenNow, enable24Hr]);
 
+  const flattenedFills = countFillsAndRemoveDuplicates(order.fills);
```

Comment:
no need to do this in this PR, but maybe these could just be derived in the provider and passed with the useOrderContext. Seen this logic twice now

Reply:
added to context